keras_hub/src/models/yolo_v8/ciou_loss.py

```diff
@@ -0,0 +1,92 @@
+import keras
+from keras import ops
+
+from keras_hub.src.bounding_box.iou import compute_ciou
+
+
+class CIoULoss(keras.losses.Loss):
```

Comment:
In KerasCV, both the `CIoULoss` and `NonMaxSuppression` were public, as they were intended to be used across different models. Currently, `YOLOV8ObjectDetector` is the only model using this loss; thus, there was no incentive to expose it at this time. Furthermore, there is no `keras_hub.losses` module where it could be appropriately exposed. However, I believe the `RetinaNet` model also uses a `NonMaxSuppression` layer that could be applied across all detectors.